### PR TITLE
Reorg: Move UI classes around

### PR DESCRIPTION
**Pull request by sbcgua**
Part of #5942, follows my comments

- all pages moved to ui_pages
  - ui_db_pages in particular were removed, merged with ui_pages
- ui_lib for common components
- ui_routing for router and services

P.S. services need some review ... deep backend uses UI packages directly.
image

### CODE REVIEW

File: src/ui/lib/package.devc.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<abapGit version="v1.0.0" serializer="LCL_OBJECT_DEVC" serializer_version="v1.0.0">
+ <asx:abap xmlns:asx="http://www.sap.com/abapxml" version="1.0">
+  <asx:values>
+   <DEVC>
+    <CTEXT>abapGit UI common library</CTEXT>
```
Comment (mbtools) [inline src/ui/lib/package.devc.xml:6]: abapGit - UI Common Library

File: src/ui/pages/package.devc.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
  <asx:abap xmlns:asx="http://www.sap.com/abapxml" version="1.0">
   <asx:values>
    <DEVC>
-    <CTEXT>abapGit - UI DB</CTEXT>
+    <CTEXT>abapGit pages</CTEXT>
```
Comment (mbtools) [inline src/ui/pages/package.devc.xml:6]: abapGit - UI Pages

File: src/ui/routing/package.devc.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<abapGit version="v1.0.0" serializer="LCL_OBJECT_DEVC" serializer_version="v1.0.0">
+ <asx:abap xmlns:asx="http://www.sap.com/abapxml" version="1.0">
+  <asx:values>
+   <DEVC>
+    <CTEXT>abapGit UI routing</CTEXT>
```
Comment (mbtools) [inline src/ui/routing/package.devc.xml:6]: abapGit - UI Routing

**Review by mbtools — CHANGES_REQUESTED** [review]
This is much better than before. Just some naming to keep our pattern (I will update descriptions of the remaining classes later)

**Comment by sbcgua** [comment]
fixed

**Review by mbtools — APPROVED** [review]
👍

**Comment by mbtools** [comment]
FYI, pulling once will leave an empty package $abapgit_ui_db. Pull again, to delete it.

**Pull request merged**